NAMESPACE

```diff
@@ -3,7 +3,6 @@
 export(count_recall_by_make)
 export(count_recall_by_model)
 export(count_recall_by_years)
-export(get_vrd_key)
 export(recall_by_make)
```

Comment:
i think we should leave `get_vrd_ke()` in the docs as the only helper function exposed to users - they can check if they have the key in their environment using this